### ISSUE TITLE
Text question

### ISSUE THREAD

**Issue by Babbosh** [Electry/VitaGrafix#87]
Will the plugin work if I put the game name beside the title id? I have many games but I mostly don't remember which one is which in the text by just looking at the titleID

example:

[PCSF00570]    "Borderlands 2"   
Enable=1
FB=960x544

**Comment by yeqing1990** [comment]
If it is not possible, it will be effective if the author is decomposed to join VitaGrafix.

**Comment by Babbosh** [comment]
Yeqing I didn't understand, sorry. Can you rephrase your last post, please?

**Comment by yeqing1990** [comment]
需要作者代码加入插件里才会有效果

**Comment by yeqing1990** [comment]
It will be effective if the author code is added to the plug-in

**Comment by Electry** [comment]
It will prob. work just fine, but I suggest you put it in a separate line with leading "#". Also, "Enable" is actually "ENABLED" (and needs to be in all caps).

Like this:

```
# Borderlands 2
[PCSF00570]
FB=640x368
```

**Comment by Babbosh** [comment]
Thanks I am gonna try now. 
Also, sorry for the misspelling, it's EnableD of course.

**Comment by gittyno1** [comment]
Isnt VitaGrafix Configurator app used so you dont need to mess with txt file?

**Comment by Babbosh** [comment]
I can confirm that it works. 
The OSD when you boot up the game instead of lasting a couple of seconds, it lasts 5/6 seconds.

Thanks again. Now text file looks neat and tidy!

**Comment by Electry** [comment]
great! And yeah, there is a configurator app you could just use instead: https://github.com/Kirezar/VitaGrafixConfigurator

:P
